src/server/cloak/index.js

```diff
@@ -90,7 +89,16 @@ module.exports = (function() {
         delete config.express;
       }
 
-      io.set('log level', config.logLevel);
+      // Cloak defaults Socket.IO log level to 1
+      io.set('log level', 1);
+
+      // Apply user Socket.IO settings
+      var ioConfig = config['socket.io'];
```

Comment:
A key like `socketIo` would be a nicer API because users wouldn't be forced to use array subscripting notation (compare `config.socketIo` and `config["socket.io"]`)